### PR TITLE
North-west Class 4 - Mya Skylar - GitHub - Week 2

### PR DESCRIPTION
**Pull request by myaskylar**
# Your Details

- Your Name: Mya Skylar
- Your City: North-west
- Your Slack Name: Mya Skylar

# Homework Details

- Module: Github
- Week: 2

### CODE REVIEW

File: homework.md [hunk 1]
```diff
@@ -92,13 +93,13 @@ A place for us to store and share our code.
...
 
-Send you code to Github
+The "commit" command is used to save your changes to the local repository
 
 ## Question 16
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+javascript
```
Comment (dharmaguadeloupe) [inline homework.md:102]: Visual Studio Code

**Review by dharmaguadeloupe — COMMENTED** [review]
Program not programming languages

**Review by konikalily — COMMENTED** [review]
Nice work Mya.

**Comment by Abed-Aghbar** [comment]
Good work Mya 👍

**Comment by Morteza-Khojasteh** [comment]
Well done Mya

**Pull request closed without merging**